extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -94,6 +98,52 @@ static public function newFromArticle( Article $article ) {
 	}
 
 	/**
+	 * Given an article represented by a Title object, return the latest comment associated with it
+	 * or return null if one can't be found.
+	 *
+	 * @param Title $title The article (or blog post) from which to find the latest comment.
+	 * @param array $param Additional parameters for this method.  Currently available keys are:
+	 *   - useSlave : A boolean value on whether to use the slave DB for the query.  If not given the master
+	 *                DB is used, with the assumption that slave lag may cause this query to fail if a comment
+	 *                was just posted.
+	 *
+	 * @return ArticleComment|null
+	 */
+	static public function latestFromTitle( Title $title, array $param = [] ) {
+		if ( empty( $param['useSlave'] ) ) {
+			$dbh = wfGetDB( DB_MASTER );
+			$flags = Title::GAID_FOR_UPDATE;
+		} else {
+			$dbh = wfGetDB( DB_SLAVE );
+			$flags = 0;
+		}
+
+		$titleText = $title->getDBkey();
+		$prefix =  $titleText . '/' . ARTICLECOMMENT_PREFIX;
```

Comment:
For joining two, possibly three things together I think simple concatenation is more readable than an implode.